certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -4173,6 +4185,658 @@
+                  "title": "Credential address",
+                  "description": "The name / address of the credential."
+                },
+                "UserName": {
+                  "type": "string",
+                  "title": "User name",
+                  "description": "The credential user name."
+                },
+                "Password": {
+                  "type": "string",
+                  "title": "Password",
+                  "description": "The credential password."
+                },
+                "CredentialType": {
+                  "type": "string",
+                  "title": "Credential type",
+                  "description": "The credential type: Windows or Generic.",
+                  "enum": [
+                    "Windows",
+                    "Generic"
+                  ]
+                },
+                "CredentialPersistence": {
+                  "type": "string",
+                  "default": "LocalMachine",
+                  "title": "Credential persistence",
+                  "description": "The credential persistence: Session (lasts just for this login session), LocalMachine (persists for this user across sessions and reboots on this computer only) or Enterprise (could roam for this across between machines).",
+                  "enum": [
+                    "Session",
+                    "LocalMachine",
+                    "Enterprise"
+                  ]
+                },
+                "SymmetricKey": {
+                  "type": "string",
+                  "title": "Symmetric encryption key",
+                  "description": "An optional symmetric encryption key used to encrypt the credential. You can only retrieve this credential if you know the key. Only usable with Generic Credentials."
+                },
+                "StorePasswordAsIdentifier": {
+                  "type": "string",
+                  "title": "Store password as identifier",
+                  "description": "If an identifier is supplied, this password will also be stored encrypted inside the Agent for later retrieval / usage by any commands which can make use of stored credentials. These passwords are only stored in memory and are lost when the Agent is closed. For example: MyExchangePassword."
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "CredentialAddress",
+                "UserName",
+                "Password",
+                "CredentialType",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Thank you for reviewing our pull request - in this scenario the command returns no output and therefore has no output schema. We distinguish between success or failure by returning the relevant HTML code, 200 (success) or 400 (failure) which will result in an exception in the Power Automate Flow. 

Most of our actions do return data and all of these have an appropriate output schema defined.

Reply:
@LukeMcGovern-UL The response code "success" should return something, at least a string with a message called "success" or "failed" which needs to be defined in the schema. An empty schema definition would not be allowed anymore.

Reply:
@sriyen-msft Ok - We have updated the Swagger definitions to return a boolean to show success or failure and any error messages in a separate string.